packages/dredd/lib/CLI.js

```diff
@@ -311,7 +311,7 @@ ${packageData.name} v${packageData.version} \
         .trim();
       logger.debug(
         'npm version:',
-        npmVersion || 'unable to determine npm version',
+        npmVersion.stdout || 'unable to determine npm version',
```

Comment:
Given the code above, we are already calling `stdout` on the process.

```js
      const npmVersion = spawnSync('npm', ['--version'])
        .stdout.toString()
        .trim();
```

Reply:
Right. So my proposed solution is probably not correct (not really tested). 

See the output in the other comment though.

Reply:
@kylef Can this be merged?